packages/react/src/Delay.tsx

```diff
@@ -1,17 +1,30 @@
-import { type PropsWithChildren, useContext, useState } from 'react'
+import { type PropsWithChildren, type ReactNode, useContext, useState } from 'react'
 import { DelayDefaultPropsContext } from './contexts'
 import { useTimeout } from './hooks'
+import { assert } from './utils'
 
 export interface DelayProps extends PropsWithChildren {
   ms?: number
+  /**
+   * @experimental This is experimental feature.
+   */
+  fallback?: ReactNode
 }
 
-export const Delay = ({ ms, children }: DelayProps) => {
+export const Delay = (props: DelayProps) => {
+  if (process.env.NODE_ENV !== 'production') {
+    if (typeof props.ms === 'number') {
+      assert(props.ms > 0, 'Delay: ms prop should be greater than or equal to 0')
+    }
+  }
```

Comment:
And I assert that ms props of Delay should be greater than or equal to 0. differently with Suspensive defaultOptions

Reply:
I made it as variable in assert.ts